blogs/nlp/LOREN/README.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Automatic Verification of Natural Language Claims
+author: Ziyue Wang
+date: 2022-4-8
+tags:
+  - Fact verafication
+  - Logic-regularized neural network
+categories:
+  - NLP
+---
+
+How to develop a model to verify a natural language statement while explaining its rationale?
+
+Reading Time: About 10 minutes.
+
+<!-- more -->
+
+Paper：<https://arxiv.org/abs/2012.13577>
+
+Github: <https://github.com/jiangjiechen/LOREN>
+
```

Comment:
Do you want to add a link to the demo as well?

Reply:
Added